ng-dev/utils/git/authenticated-git-client.ts

```diff
@@ -123,6 +123,25 @@ export class AuthenticatedGitClient extends GitClient {
         'Unable to configure `AuthenticatedGitClient` as it has been configured already.',
       );
     }
+    if (process.env['GITHUB_ACTIONS']) {
```

Comment:
I like the `configureForGithubActions` call for now, but I think long-term (or when you feel like doing it right now), I think we should create a wrapper in the shared github action utils that just creates an instance of the `AuthenticatedGitClient`. It feels wrong integrating this special Github action logic in the `ng-dev` command tool.. especially beause right now I would not be able to run a `yarn ng-dev` command in an action right?

Reply:
Agreed, we should do something more of a wrapper that properly handles it, I wanted to put this catch in so if we accidentally try to rely on `configure` during a Github action it would tell us it is wrong.

Reply:
yeah, that definitely make sense. Might be a good thing to enforce through a linter or through Bazel target visibility